### PR TITLE
Only show current base shipments for multibase users

### PR DESCRIPTION
**Pull request by fhenrich33**

### CODE REVIEW

**Comment by fhenrich33** [comment]
[2.0 FE Bug: Fix ShipmentOverviewView for MultiBase Users](https://trello.com/c/4ILUozSB/1332-20-fe-bug-fix-shipmentoverviewview-for-multibase-users)

File: front/src/views/Transfers/ShipmentsOverview/ShipmentsOverviewView.tsx [hunk 1]
```diff
@@ -27,80 +29,82 @@ function ShipmentsOverviewView() {
...
   const graphqlToTableTransformer = (shipmentQueryResult: ShipmentsQuery | undefined) =>
-    shipmentQueryResult?.shipments.map((element) => {
-      if (globalPreferences?.availableBases) {
-        const availableBaseIds = globalPreferences.availableBases.map((base) =>
-          parseInt(base.id, 10),
-        );
-        const sourceBaseId = parseInt(element.sourceBase.id, 10);
-        const targetBaseId = parseInt(element.targetBase.id, 10);
+    shipmentQueryResult?.shipments
+      .filter((shipment) => shipment.sourceBase.id === baseId)
```
[the remark this answers is not in the document]
Comment (pylipp) [reply front/src/views/Transfers/ShipmentsOverview/ShipmentsOverviewView.tsx:33]: @fhenrich33 I think you want to display all shipments with `targetBase.id === baseId`, i.e. towards the user's current base, too.

File: front/src/views/Transfers/ShipmentsOverview/ShipmentsOverviewView.tsx [hunk 2]
```diff
@@ -30,7 +30,7 @@ function ShipmentsOverviewView() {
   // transform shipments data for UI
   const graphqlToTableTransformer = (shipmentQueryResult: ShipmentsQuery | undefined) =>
     shipmentQueryResult?.shipments
-      .filter((shipment) => shipment.sourceBase.id === baseId)
+      .filter((shipment) => shipment.sourceBase.id === baseId && shipment.targetBase.id === baseId)
```
Comment (pylipp) [inline front/src/views/Transfers/ShipmentsOverview/ShipmentsOverviewView.tsx:33]: Sorry, I didn't clarify; you need || here

Comment (fhenrich33) [reply front/src/views/Transfers/ShipmentsOverview/ShipmentsOverviewView.tsx]: That's actually my bad, thanks for the catch!

**Review by pylipp — APPROVED** [review]

**Pull request merged**